daemon/execdriver/windows/run.go

```diff
@@ -223,10 +224,31 @@ func (d *Driver) Run(c *execdriver.Command, pipes *execdriver.Pipes, hooks execd
 
 	configuration := string(configurationb)
 
-	err = hcsshim.CreateComputeSystem(c.ID, configuration)
-	if err != nil {
-		logrus.Debugln("Failed to create temporary container ", err)
-		return execdriver.ExitStatus{ExitCode: -1}, err
+	// The following a workaround for Windows TP4 which has a networking
+	// bug which fairly frequently returns an error. Back off and retry.
```

Comment:
would be great to add a `TODO(windows)` to make sure this hack gets removed in time :)